evap/evaluation/tests/test_views.py

```diff
@@ -246,3 +248,44 @@ def test_notebook(self):
 
         user.refresh_from_db()
         self.assertEqual(user.notes, self.note)
+
+
+class TestResetEvaluation(WebTestStaffMode):
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.manager = make_manager()
+        cls.semester = baker.make(Semester, results_are_archived=True)
```

Comment:
is `results_are_archived` relevant? If so, maybe add a short comment why? I find it a bit confusing because on a real instance, you wouldn't want to change evaluation states of archived evaluations / in archived semesters

Reply:
I do not think this is relevant. The original commit was probably in January or something